Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -170,5 +170,68 @@ public void RefreshErrorMessage_WithInternalAndRowFailures_ReturnsAppendedMessag
 
       Assert.That(validator.ErrorMessage, Is.EqualTo("Errors on BocList\r\nErrors on rows in BocList\r\n"));
     }
+
+    [Test]
+    public void EvaluateIsValid_WithoutFailures_EmptyErrorMessageAndValid ()
+    {
+      var validationFailureRepoStub = new Mock<IBocListValidationFailureRepository>();
+      validationFailureRepoStub.Setup(_ => _.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(false))
+          .Returns(Array.Empty<BocListValidationFailureWithLocationInformation>());
+
+      var error = "BocList has validation errors.";
+      var resourceManagerStub = new Mock<IResourceManager>();
+      resourceManagerStub.Setup(_ => _.TryGetString("Remotion.ObjectBinding.Web.UI.Controls.BocList.RemainingValidationFailureText", out error)).Returns(true);
+
+      var bocListMock = new Mock<Control> { CallBase = true };
+      bocListMock.As<IBocList>().Setup(_ => _.ValidationFailureRepository).Returns(validationFailureRepoStub.Object);
+      bocListMock.As<IBocList>().Setup(_ => _.GetResourceManager()).Returns(resourceManagerStub.Object);
```

Comment:
Depending on the production code, you could also set this up with a `Throws(new  InvalidOperationException("GetResourceManager() should be called in this test.")` to avoid needing a complete stubbing.